ajax/libs/reframe.js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "reframe.js",
+  "filename": "reframe.min.js",
+  "version": "2.1.18",
+  "description": "Reframe.js: responsive iframes for embedded content",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dollarshaveclub/reframe.js.git"
+  },
+  "keywords": [
+    "js",
+    "iframes",
+    "fluid",
+    "responsive",
+    "video",
+    "fitvids",
+    "youtube",
+    "embeds",
+    "padding-top"
+  ],
+  "author": "Jeff Wainwright <jjwainwright2@gmail.com> (https://jeffry.in)",
+  "homepage": "https://dollarshaveclub.github.io/reframe.js/",
+  "npmName": "reframe.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(*.scss || *.es.js)"
```

Comment:
`"**/!(*.scss || *.es.js)"` not `"**/!(*.scss|*.es.js)"`, would you please test the rules locally?